handler/strings.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 var (
-	archRe     = regexp.MustCompile(`(arm64|arm|386|amd64|32|64)`)
+	archRe     = regexp.MustCompile(`(arm64|arm|386|amd64|32|64|x86_64|aarch64)`)
```

Comment:
can you plz move these to the front? otherwise it'll return "64"

Reply:
I did try it here[1] beforehand it seemed to macht the longest, but I can change regardless.

[1] https://regexr.com/77j6n

Reply:
Oh interesting, TIL
will test in Go soon and confirm